susemanager/src/mgr_bootstrap_data.py

```diff
@@ -1470,6 +1495,11 @@
         'DEST' : '/srv/www/htdocs/pub/repositories/debian/11/bootstrap/',
         'TYPE' : 'deb'
     },
+    'debian12-amd64' : {
+        'PDID' : ['TODO', 'TODO'], 'BETAPDID' : ['TODO'], 'PKGLIST' : PKGLISTDEBIAN12,
```

Comment:
I guess these TODO needs to be replaced before merging the PR?

Reply:
Yes, someone needs to allocate free IDs

Reply:
This is not just free IDs. We need to setup something for it. Please remove this from this PR.
We need to add it when we have that thing done. It is not directly required for Uyuni. This is more used in SUSE Manager

Reply:
Removed that section